.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: vim
+language: ruby
```

Comment:
I think Ruby its the default for unknown languages: http://docs.travis-ci.com/user/getting-started/#Note